lib/idx/remediate.ts

```diff
@@ -48,29 +48,15 @@ export function getRemediator(
       
     const T = remediators[remediation.name];
     remediator = new T(remediation, values);
-    if (flowMonitor.isRemediatorCandidate(remediator, idxRemediations, values)) {
```

Comment:
how would we filter out remediations that are valid within the `idxResponse`, but don't match the desired flow?

I had to do this for `idx.unlockAccount`:
If I called `idx.unlockAccount({username: 'foo', authenticator: '...'})`, `auth-js` would attempt to auto-remediate `/identify`, because `username` was present. To prevent this, I added:
```javascript
  isRemediatorCandidate(remediator, remediations?, values?) {
    const prevRemediatorName = this.previousRemediator?.getName();
    const remediatorName = remediator.getName();

    // required to prevent /identify from auto-remediating when 'username' passed
    if (remediatorName === 'identify' && !prevRemediatorName) {
      return false;
    }
....
```
https://github.com/okta/okta-auth-js/pull/1030/files#diff-f90ddef32a0b8883f13e83cb687f25669055748ac81a9b5355319dbf5475ca9cR21

Reply:
I was hoping to use `isRemediatorCandidate()`  check for picking next remediation after polling:
```
- enroll-poll
- select-enrollment-channel <-
```
Is there a suggested way to pick non-first remediation from the list?